js/adapt-contrib-tutor.js

```diff
@@ -2,11 +2,34 @@ define([
     'coreJS/adapt'
 ],function(Adapt) {
 
-    Adapt.on('questionView:showFeedback', function(view) {
+      /**
+       * The screen width can fall into one of three categories: large, medium
+       * or small. There is a large and a small image. If the screen width is
+       * medium get the small image src otherwise get the src corresponding to
+       * the screen width.
+       * @param image
+       * @param width
+       * @returns {*}
+       */
+      function getImageSrc (image, width) {
+        if (width !== 'large') {
```

Comment:
Could you add `if (image.src) return image.src;` above this line? 

That would allow for people using the framework directly to specify just a `src` property if they don't want different images for large and small screen sizes - as per other components that use a graphic (e.g. graphic component).

Reply:
Equally you could achieve the same thing by adding `if(imageSrc)` to line 26